content/F2015/F15-event-orientation.md

```diff
@@ -0,0 +1,25 @@
+title: Orientation Event and Dinner
+Date: 2015-09-13 15:00-18:00
+Category: Events
+Tags: social, big csters, academic
+Slug: orientation-F15
+Author: Fatema Boxwala
+Summary: Come out and learn about the university life, and get to know some
```

Comment:
Summary/other metadata all has to be all on one line to be processed properly; this text will render extraneously at the top of the doc.